### PR TITLE
Feature/add login

### PR DESCRIPTION
**Pull request by IgorGabrielNunes**
Inserção da footer e ajuste da cadastro.

### CODE REVIEW

File: db/db.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
         "Cheddar",
         "Bacon"
       ],
-      "status": "Em produção",
```
Comment (julianostroschon) [inline db/db.json:99]: The burgers do not need to come to db, or it have to go to another file.
A file (db.json) have the information to construct the Burgers.
And another file (req.json) are constructed only with de burgers requests. That way, the req.json do not need to be submited to repo

File: src/components/CadastroLogin.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
+<template>
+  <form id="cadastro-form" @submit="createCadastro">
+    <div class="input-container">
+      <label for="nome">Nome Completo:</label>
+      <input
+        type="text"
+        id="nomecompleto"
+        nome="nomecompleto"
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove this atribute of all input tags

File: src/components/CadastroLogin.vue [hunk 3]
```diff
@@ -0,0 +1,154 @@
+<template>
+  <form id="cadastro-form" @submit="createCadastro">
+    <div class="input-container">
+      <label for="nome">Nome Completo:</label>
+      <input
+        type="text"
+        id="nomecompleto"
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove id atribute of all input tags

File: src/components/CadastroLogin.vue [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    </div>
+
+    <div class="input-container">
+      <input type="submit" class="submit-btn" value="Cadastrar" />
+    </div>
+  </form>
+</template>
+
+<script>
+import Cadastro from "../views/Cadastro.vue";
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove it

File: src/components/CadastroLogin.vue [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+export default {
+  components: { Cadastro },
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove

File: src/components/CadastroLogin.vue [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        type="text"
+        id="nomeuser"
+        nome="nomeuser"
+        v-model="nomeuser"
+        placeholder="Digite seu nome de usuario!"
+      />
+    </div>
+
+    <div class="input-container">
+      <input type="submit" class="submit-btn" value="Cadastrar" />
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: @click.prevent="createCadastro"

File: src/components/CadastroLogin.vue [hunk 7]
```diff
@@ -0,0 +1,154 @@
+<template>
+  <form id="cadastro-form" @submit="createCadastro">
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove ( @submit="createCadastro" )

File: src/components/CadastroLogin.vue [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+       });
+
+    },
+  }
+}
+</script>
+
+<style scoped>
+#cadastro-form {
+  max-width: 300px;
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue:103]: Never use 'px' with a units to define a size, It's not responsive

File: src/components/Footer.vue [hunk 9]
```diff
@@ -1,6 +1,52 @@
 <template>
   <footer id="footer">
-    <span>Desenvolvimento: Departamento de TI - Fundimisa © 2020</span>
+    <img src="../../public/logo-branco.png" />
+    <span class="span-ceredits">Desenvolvimento: Departamento de TI - Fundimisa © 2020</span> <br>
```
Comment (julianostroschon) [inline src/components/Footer.vue]: 2020??????
We already in 2022

File: src/components/CadastroLogin.vue (hunk 7 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove submit

File: src/components/CadastroLogin.vue (hunk 3 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: remove:
id
nome

to all input tags

File: src/components/CadastroLogin.vue (hunk 6 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: include @click.prevent="function to create"

File: src/components/CadastroLogin.vue (hunk 4 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove

File: src/components/CadastroLogin.vue (hunk 5 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Remove

File: src/components/CadastroLogin.vue [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+        nomecompleto: this.nomecompleto,
+        email: this.email,
+        senha: this.senha,
+        nomeuser: this.nomeuser,
+        status: "Criado!"
+      }
+
+    const dataJson = JSON.stringify(data);
+       
+        const req = await fetch("http://localhost:3000/Usuarios", {
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: Use the same way to call all routes, if in lower case, pascal, or another case. But use the same to all project

File: src/components/CadastroLogin.vue (hunk 8 above)
Comment (julianostroschon) [inline src/components/CadastroLogin.vue:103]: Do not use 'px' unit to define a configuration, it's not responsive unit. Think about '%', 'rem', 'vh or vw'...

File: src/components/CadastroLogin.vue [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+select {
+  padding: 5px 10px;
+  width: 300px;
+  margin: auto;
+}
+
+.submit-btn {
+  background-color: rgb(255, 0, 0);
+  color: #ffffff;
+  font-weight: bodl;
```
Comment (julianostroschon) [inline src/components/CadastroLogin.vue]: I guess thats can be (bold), thats appears to me like an error

File: src/components/Dashboard.vue [hunk 12]
```diff
@@ -7,7 +7,9 @@
         <div>Pão:</div>
         <div>Carne:</div>
         <div>Opcionais:</div>
-        <div>Ações:</div>
+        <div>Ações:
+
```
Comment (julianostroschon) [inline src/components/Dashboard.vue:11]: Why the white space???
Use a single form to program in all project

File: src/components/Footer.vue (hunk 9 above)
Comment (julianostroschon) [inline src/components/Footer.vue]: why 2020???

We already in 2022

File: src/components/Footer.vue [hunk 13]
```diff
@@ -1,6 +1,52 @@
...
+    <div class="footer-box"> 
+      <div id="sow-editor-3" class="footer-widget">
+        <div class="swidget-editor-base">
+          <div class="siteorigin-widget">
+            <h3 style="text-align: right;">
```
Comment (julianostroschon) [inline src/components/Footer.vue]: Do not use 'style' inline, put a class or an id to stylize on '<style>' tag. That way do you put all style information on tag used to do this

File: src/components/Footer.vue [hunk 14]
```diff
@@ -1,6 +1,52 @@
...
+              <span style="text-align: right;">Localização:</span>
```
Comment (julianostroschon) [inline src/components/Footer.vue]: Same above

File: src/components/Footer.vue [hunk 15]
```diff
@@ -1,6 +1,52 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="footer-box1"> 
+      <div id="sow-editor-2" class="footer-widget1">
+        <div class="swidget-editor-base1">
+          <div class="siteorigin-widget1">
+            <h3 style="text-align: left;">
```
Comment (julianostroschon) [inline src/components/Footer.vue]: Do not use style inline

File: src/components/Footer.vue [hunk 16]
```diff
@@ -1,6 +1,52 @@
...
+              CEP:98.855-000
+              <br>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="footer-box1"> 
+      <div id="sow-editor-2" class="footer-widget1">
```
Comment (julianostroschon) [inline src/components/Footer.vue]: a Lot of classes unuseds on <style>, to me that can be removed. Do not?

File: src/components/Footer.vue [hunk 17]
```diff
@@ -16,10 +62,18 @@ export default {
...
   align-items: center;
   justify-content: center;
   
 }
+
+#sow-editor-3{
+   text-align: right;
+}
+
+.span-ceredits {
```
Comment (julianostroschon) [inline src/components/Footer.vue]: I guess do you mean credits

File: src/components/Footer.vue [hunk 18]
```diff
@@ -16,10 +62,18 @@ export default {
...
   background-color: #222;
   color: #fff;
-  text-align: center;
   display: flex;
   align-items: center;
   justify-content: center;
   
 }
+
+#sow-editor-3{
```
Comment (julianostroschon) [inline src/components/Footer.vue]: sow or show????

Why do you use it?

File: src/components/LoginPage.vue [hunk 19]
```diff
@@ -0,0 +1,89 @@
+<template>
+    <form id="login-form" @submit="loginForm">
```
Comment (julianostroschon) [inline src/components/LoginPage.vue:2]: Use button to submit, remove this function and add to the button

File: src/components/LoginPage.vue [hunk 20]
```diff
@@ -0,0 +1,89 @@
...
+
+      <div class="input-container">
+      <label for="nome">Digite seu email:</label>
```
Comment (julianostroschon) [inline src/components/LoginPage.vue]: Why atribute 'for'?????

File: src/components/LoginPage.vue [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+      <input
+        type="text"
+        id="nomecompleto"
```
Comment (julianostroschon) [inline src/components/LoginPage.vue]: If you do not use 'id' and 'nome' atribute, remove them

File: src/components/LoginPage.vue [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+        nome="nomecompleto"
+        v-model="email"
+        placeholder="Digite seu email!"
+      />
+      </div>
+      
+      <div class="input-container">
+      <label for="email">Digite sua senha::</label>
```
Comment (julianostroschon) [inline src/components/LoginPage.vue]: for, again. Remove it

File: src/components/LoginPage.vue [hunk 23]
```diff
@@ -0,0 +1,89 @@
...
+      </div>
+
+      <div class="input-container">
+      <input type="submit" class="submit-btn" value="Login" />
+    </div>
+    </form>
+</template>
+
+ <script>
+ import Login from "../views/Login.vue"
```
Comment (julianostroschon) [inline src/components/LoginPage.vue]: If you do not use an imported file. You do not have to import it

File: src/components/LoginPage.vue [hunk 24]
```diff
@@ -0,0 +1,89 @@
...
+ 
+ export default {
+   components: Login,
```
Comment (julianostroschon) [inline src/components/LoginPage.vue]: Remove, because you do not use it

File: src/components/LoginPage.vue [hunk 25]
```diff
@@ -0,0 +1,89 @@
...
+       email : null,
+       senha: null
+     }
+   }
+ }
+ </script>
+
+ <style scoped>
+#login-form {
+  max-width: 300px;
```
Comment (julianostroschon) [inline src/components/LoginPage.vue:44]: Same of another file. Please do not use 'px' unit, or if do you use in this project, do not use on real project of Elyte page

File: src/components/Navbar.vue [hunk 26]
```diff
@@ -5,6 +5,8 @@
     </router-link>
     <router-link to="/">Home</router-link> 
     <router-link to="/pedidos">Pedidos</router-link>
+    <router-link to="/Cadastro">Cadastro</router-link>
```
Comment (julianostroschon) [inline src/components/Navbar.vue]: The routes are defined on lower case, in our projects

File: src/router/index.js [hunk 27]
```diff
@@ -1,6 +1,8 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import Home from '../views/Home.vue'
 import Pedidos from '../views/Pedidos.vue'
+import Cadastro from "../views/Cadastro.vue"
```
Comment (julianostroschon) [inline src/router/index.js]: You can use @, it is an alias to src/ folder, to do not use '../' reference

File: src/router/index.js [hunk 28]
```diff
@@ -12,6 +14,16 @@ const routes = [
...
     name: 'pedidos',
     component: Pedidos
+  },
+  {
+    path: '/cadastro',
+    name: 'cadastro',
+    component: Cadastro
+  },
+  {
+    path: '/login',
```
Comment (julianostroschon) [inline src/router/index.js:24]: Your routes are defined in lowercase, but the use of them with a another case

**Review by julianostroschon — CHANGES_REQUESTED** [review]

File: src/components/CadastroLogin.vue (hunk 3 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: Removid.

File: src/components/CadastroLogin.vue (hunk 2 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: Removid.

File: src/components/CadastroLogin.vue (hunk 5 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removid.

File: src/components/CadastroLogin.vue (hunk 6 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: Added

File: src/components/CadastroLogin.vue (hunk 7 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removed.

File: src/components/CadastroLogin.vue (hunk 8 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue:103]: ok, removid.

File: src/components/Footer.vue (hunk 9 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: removid.

File: src/components/CadastroLogin.vue (hunk 7 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removid

File: src/components/CadastroLogin.vue (hunk 3 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removid

File: src/components/CadastroLogin.vue (hunk 6 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: incluide.

File: src/components/CadastroLogin.vue (hunk 5 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removid

File: src/components/CadastroLogin.vue (hunk 4 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: removed.

File: src/components/CadastroLogin.vue (hunk 10 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: Resolved.

File: src/components/CadastroLogin.vue (hunk 8 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue:103]: resolved.

File: src/components/CadastroLogin.vue (hunk 11 above)
Comment (IgorGabrielNunes) [reply src/components/CadastroLogin.vue]: resolved.

File: src/components/Footer.vue (hunk 9 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: resolved.

File: src/components/Footer.vue (hunk 13 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: Resolved.

File: src/components/Footer.vue (hunk 14 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: Resolved.

File: src/components/Footer.vue (hunk 15 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: resolved.

File: src/components/Footer.vue (hunk 16 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: resolved.

File: src/components/Footer.vue (hunk 17 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: yes, resolved.

File: src/components/Footer.vue (hunk 18 above)
Comment (IgorGabrielNunes) [reply src/components/Footer.vue]: removed.

File: src/components/LoginPage.vue (hunk 19 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue:2]: Resolved.

File: src/components/LoginPage.vue (hunk 20 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue]: Removed.

File: src/components/LoginPage.vue (hunk 21 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue]: removed all.

File: src/components/LoginPage.vue (hunk 22 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue]: removed.

File: src/components/LoginPage.vue (hunk 23 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue]: removed.

File: src/components/LoginPage.vue (hunk 24 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue]: removed.

File: src/components/LoginPage.vue (hunk 25 above)
Comment (IgorGabrielNunes) [reply src/components/LoginPage.vue:44]: I will only use this project, next I will take care of it more.

File: src/components/Navbar.vue (hunk 26 above)
Comment (IgorGabrielNunes) [reply src/components/Navbar.vue]: Ajusted.

File: src/router/index.js (hunk 27 above)
Comment (IgorGabrielNunes) [reply src/router/index.js]: Ajusted.

File: src/router/index.js (hunk 28 above)
Comment (IgorGabrielNunes) [reply src/router/index.js:24]: Ajusted.

File: db/db.json (hunk 1 above)
Comment (IgorGabrielNunes) [reply db/db.json:99]: RESOLVED!

**Comment by IgorGabrielNunes** [comment]
RESOLVED ALL ERRORS!

File: .zshrc [hunk 29]
```diff
@@ -31,3 +31,15 @@ export NVM_DIR="$HOME/.nvm"
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+export NVM_DIR="$HOME/.nvm"
```
Comment (julianostroschon) [inline .zshrc:34]: Please remove this file, and add a rule on .gitignore to do not send it to repo again

File: src/App.vue [hunk 30]
```diff
@@ -15,7 +15,7 @@ export default {
   },
   data() {
     return {
-      logo_src:"../logo.png",
+      logo_src:"../faviconelyte.ico",
```
Comment (julianostroschon) [inline src/App.vue:18]: It's a logo or a favicon???

File: src/App.vue [hunk 31]
```diff
@@ -15,7 +15,7 @@ export default {
...
       app_name: "Make your Burguer"
```
Comment (julianostroschon) [inline src/App.vue:19]: Change the name to a singular app, not the same of class

**Review by julianostroschon — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 30 above)
Comment (IgorGabrielNunes) [reply src/App.vue:18]: yes, I used logofavicon and logo, for both parties.

**Pull request closed without merging**